docs/guides/auth.md

```diff
@@ -153,12 +153,12 @@ described above appear in each language: more languages are coming soon.
 
 ```ruby
 # Base case - No encryption
-stub = Helloworld::Greeter::Stub.new('localhost:50051')
+stub = Helloworld::Greeter::Stub.new('localhost:50051', :this_channel_is_insecure)
 ...
 
 # With server authentication SSL/TLS
 creds = GRPC::Core::Credentials.new(load_certs)  # load_certs typically loads a CA roots file
-stub = Helloworld::Greeter::Stub.new('localhost:50051', creds: creds)
+stub = Helloworld::Greeter::Stub.new('localhost:50051', creds)
```

Comment:
please use 'greeter.googleapis.com' as opposed to 'localhost:50051' which would most certainly need an ssl name override.